sycl/test-e2e/AOT/fpga-aoc-archive.cpp

```diff
@@ -0,0 +1,28 @@
+// Test compiler behaviors for -fintelfpga with -fsycl-link.
```

Comment:
Do we really need this test? It seems to be just a subset of the 'severe' test?

Reply:
That's true.  The severe case covers what this does as well.  I'll clean up and leave the severe case.